### ISSUE TITLE
make latex failed

### ISSUE THREAD

**Issue by choeffer** [sphinx-doc/sphinx#5563]
Subject: used "make latex"

### Problem

make latex command fails

#### Procedure to reproduce the problem

```
(venv) pink@debian-vm:~/Dokumente/csalgori/docs$ make latex
Sphinx v1.8.1 in Verwendung
Erstelle Ausgabeverzeichnis…
loading intersphinx inventory from https://docs.python.org/objects.inv...
intersphinx inventory has moved: https://docs.python.org/objects.inv -> https://docs.python.org/3/objects.inv
building [mo]: targets for 0 po files that are out of date
building [latex]: all documents
updating environment: 2 added, 0 changed, 0 removed
reading sources... [100%] index                                                 
looking for now-outdated files... none found
pickling environment... erledigt
checking consistency... erledigt
processing 360pibot.tex...index api 
resolving references...
```

#### Error logs / results

```
Exception occurred:
  File "/home/pink/Dokumente/csalgori/venv/lib/python3.5/site-packages/docutils/nodes.py", line 567, in __getitem__
    return self.attributes[key]
KeyError: 'docname'
The full traceback has been saved in /tmp/sphinx-err-csfzt_i1.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Makefile:19: die Regel für Ziel „latex“ scheiterte
make: *** [latex] Fehler 2
```

#### Expected results

*.tex file of the documentation

### Reproducible project / your project

the error log is also included in docs/error.log

[docs.zip](https://github.com/sphinx-doc/sphinx/files/2511058/docs.zip)

[requirements.txt](https://github.com/sphinx-doc/sphinx/files/2511062/requirements.txt)

[python_scripts.zip](https://github.com/sphinx-doc/sphinx/files/2511066/python_scripts.zip)

### Environment info
- OS: Debian Stable
- Python version: 3.5
- Sphinx version: 1.8.1
- texlive, texlive-latex-extra, texworks

**Comment by tk0miya** [comment]
Confirmed:
```
# Dockerfile
FROM tk0miya/sphinx-html

RUN curl -LO https://github.com/sphinx-doc/sphinx/files/2511058/docs.zip
RUN curl -LO https://github.com/sphinx-doc/sphinx/files/2511062/requirements.txt
RUN curl -LO https://github.com/sphinx-doc/sphinx/files/2511066/python_scripts.zip
RUN unzip docs.zip
RUN unzip python_scripts.zip
RUN pip3 install -r requirements.txt
RUN make -C docs latex
```

It seems some extension generates a footnote_reference node not having docname. I think it is a bug of Sphinx Transform.

**Comment by tk0miya** [comment]
Fixed by #5580.
Thank you for reporting.
